styles/typography.less

```diff
@@ -18,3 +18,40 @@ a {
     color: @primary;
   }
 }
+
+/* Font-size for pages' h1 header; overwriting Bootstrap */
+
+div.page-header h1 {
```

Comment:
Don't prefix the style with an element name, in case that element needs to change in the future. `h1` is ok, though, since I think it's how Bootstrap applies similar styles. (here and below)

Reply:
Are you sure? Bootstrap should be loaded first, so styles written in the theme should override them if they have the same specificity. It may be the styles in `/pages/page-common.less` that you're battling, in which case you can overwrite those.

Reply:
Pretty sure. Checked this one already. Will think on this.

Reply:
Thanks for the hint. I've placed the changes there.